bin/analyze_hoc_activity

```diff
@@ -88,7 +88,25 @@ def main()
     hoc_company_totals: hoc_company_totals,
   }
 
-  lines_of_code = rack_env?(:production) ? `#{deploy_dir('bin','count-lines-of-code-from-milestone-logs_v2')}`.to_i : 0
+  # This default number from 2016-01-04.
+  DEFAULT_LINES_OF_CODE = 11151730618
+  # In the production environment, begin with a known previous value in case we
+  # are unable to obtain a more recent number.
+  lines_of_code = rack_env?(:production) ? DEFAULT_LINES_OF_CODE : 0
+  # Since s3 (which the binary relies on) may fail, we retry a few times hoping
+  # for success.
+  if rack_env?(:production)
+    3.times do
+      # Note that by executing via backticks, the global variable $? is set.
+      lines_of_code_from_milestone = `#{deploy_dir('bin','count-lines-of-code-from-milestone-logs_v2')}`.to_i
+      # If we received a real number (larger than from 2016-01-04) and a proper
+      # status, do not continue retrying.
+      if lines_of_code_from_milestone >= DEFAULT_LINES_OF_CODE && $?.exitstatus == 0
```

Comment:
Could you log a warning for the broken state where the call succeeds but lines_of_code_from_milestone < DEFAULT_LINES_OF_CODE?

This would be a sign of a regression that we need to investigate.